docs/jest-integration.md

```diff
@@ -87,6 +87,58 @@ which can be specified through the CLI client.
 }
 ```
 
+## Enabling TypeScript Jest tests
+
+Jest supports execution of tests written in other languages than JavaScript via
+dedicated extensions. Probably most prominent is the TypeScript support, which
+can be enabled via [`ts-jest`](https://kulshekhar.github.io/ts-jest/).
+
+Describing all available configuration options is out of scope for this
```

Comment:
"Describing all available configuration options is out of scope for this
documentation, and we assume you already set up your TypeScript project."  This can be cut---what's not here is out of scope.

Reply:
Removed.